Cargo.toml

```diff
@@ -25,6 +25,7 @@ serde_json = { version = "=1.0.69", default-features = false, features = ["alloc
 embedded-time = "0.12.1"
 arbitrary = { version = "0.4.7", features = ["derive"], optional = true }
 rand = { version = "0.8.4", optional = true }
+ed25519-compact = { version = "1", default-features = false, optional = true }
```

Comment:
As far as I can tell, `opt_size` feature is not yet released in latest version 1.0.11, and only present on the tip of master branch on https://github.com/jedisct1/rust-ed25519-compact/tree/master
When I replace version 1.0.11 with current github version https://github.com/jedisct1/rust-ed25519-compact#26992216
then sizes are, with `opt_size`:
<pre>
 5.5%  13.7%  19.6KiB ed25519_compact
</pre>
and without `opt_size`:
<pre>
 3.5%   9.1%  12.2KiB ed25519_compact
</pre>

Reply:
Awesome! Thanks for testing. So it seems we're better without :)

Reply:
Oh. Terribly sorry, my bad

It's vice versa, 12.2 KiB _with_ `opt_size`, 19.6KiB _without_ `opt_size`

So using `opt_size` does make sense, but after it will get into released version, I guess.